### PR TITLE
Run workspace member's tests

### PR DESCRIPTION
**Pull request by jackcmay**
#### Problem

Workspace member test's were are not being run in automation

#### Summary of Changes

Fixes #

### CODE REVIEW

**Comment by garious** [comment]
Why not make this generic like in the lines just above? https://github.com/solana-labs/solana/pull/1666/files#diff-89aed6b2152a9d6eccdec600814193c8R27

**Comment by jackcmay** [comment]
Yeah, could do that, unlike /tests or /bins which cargo pretty much mandates that they are part of the workspace, other members have to be listed explicitly in the Cargo.toml file so I was following that model.  Good suggestion though, I'll prob do that.

File: ci/test-stable-perf.sh [hunk 1]
```diff
@@ -30,6 +30,13 @@ for test in tests/*.rs; do
   _ cargo test --verbose --jobs=1 --features="$FEATURES" --test="$test"
 done
 
+echo --- "( cd programs/native/bpf_loader ; cargo test --verbose )"
```
Comment (mvines) [inline ci/test-stable-perf.sh]: Running these in ci/test-stable.sh should be sufficient.   These programs have no cuda, erasure, chacha in them

File: ci/test-stable.sh [hunk 2]
```diff
@@ -23,6 +23,13 @@ for test in tests/*.rs; do
   _ cargo test --verbose --jobs=1 --test="$test"
 done
 
+echo --- "( cd programs/native/bpf_loader ; cargo test --verbose )"
```
Comment (mvines) [inline ci/test-stable.sh]:
```bash
for program in programs/native/*; do
  echo --- $program
  (
    set -x
    cd $program
    cargo test --verbose
  )
done
```

**Pull request merged**